src/app.js

```diff
@@ -1,6 +1,10 @@
 import 'audioworklet-polyfill';
 import './shims';
-import './audiocontext';
+
+import { getAudioContext, userStartAudio } from './audiocontext';
+p5.prototype.getAudioContext = getAudioContext;
+p5.prototype.userStartAudio = userStartAudio;
```

Comment:
looks like this includes changes from #508, best to keep the changes in separate PR's if possible (or put them all in the same PR)

Reply:
Thats why i kept it for review after 508 😅